docs/sources/tasks/debug.md

```diff
@@ -92,6 +92,7 @@ Live debugging is not yet available in all components.
 
 Supported components:
 * prometheus.relabel
+* otelcol.processor.*
```

Comment:
It'd be good to also document the performance penalty on `otelcol` components.

Reply:
with the latest change, the performance penalty only appears when a stream is open. This is expected and does not need additional documentation